### PR TITLE
Convert README.md to README.rst

### LINKED ISSUE #34: Convert README.md into README.rst
`rst` documentation format is used heavily in the rest of this module and this module adds directives to `rst`. It probably makes sense for the README to also be in `rst` format too.

### PR DESCRIPTION
**Pull request by daniellimws**
Close #34 

Signed-off-by: Daniel Lim Wee Soong <weesoong.lim@gmail.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 from sphinxcontrib_verilog_diagrams import __version__
 
-readme_file = path.join(path.dirname(path.abspath(__file__)), 'README.md')
+readme_file = path.join(path.dirname(path.abspath(__file__)), 'README.rst')
```
Comment (mithro) [inline setup.py:27]: You need to update the `long_description_content_type` value in `setup.py`

**Comment by mithro** [comment]
Can you rebase this change?

File: setup.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 else:
     __version__ = "0.0.dev0"
 
-readme_file = path.join(__dir__, 'README.md')
+readme_file = path.join(__dir__, 'README.rst')
```
Comment (mithro) [inline setup.py:35]: On line 54, you need to update the `long_description_content_type` value.

Comment (daniellimws) [reply setup.py:35]: What should I change it to?

Comment (mithro) [reply setup.py:35]: https://packaging.python.org/specifications/core-metadata/#description-content-type-optional

Comment (daniellimws) [reply setup.py:35]: Oh sorry I was looking at the wrong thing, thought you wanted me to change `description` on line 54.

Comment (mithro) [reply setup.py:35]: GitHub doesn't let you put comments on unmodified lines it seems :-(

**Pull request merged**